artview/components/select_region.py

```diff
@@ -173,39 +172,52 @@ def button_press_callback(self, event):
                                        marker='o')
                     self.previous_point = [x, y]
```

Comment:
See how your image editor work, those softwares are the specialists in this functionality.